### PR TITLE
Added Helix Installation Instructions to README.md

### PR DESCRIPTION
**Pull request by samusz**
As discussed in #993. (Sorry for the delay in creating this PR)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,27 @@ For `eglot`, use:
...
+scope = "source.elixir"
+injection-regex = "elixir"
+file-types = ["ex", "exs"]
+roots = ["mix.exs"]
+auto-format = true
+diagnostic-severity = "Hint"
+comment-token = "#"
+indent = {tab-width = 2, unit = " "}
+language-server = {command = "elixir-ls"}
+```
```
Comment (lukaszsamson) [inline README.md:115]: Please add some example config, basing on https://elixirforum.com/t/helix-editor-for-elixir-development/54964/24?u=lukaszsamson it's done like that
```
config = { elixirLS.dialyzerEnabled = false }
```

File: README.md [hunk 2]
```diff
@@ -92,6 +92,27 @@ For `eglot`, use:
...
+name = "elixir"
+scope = "source.elixir"
+injection-regex = "elixir"
+file-types = ["ex", "exs"]
+roots = ["mix.exs"]
+auto-format = true
+diagnostic-severity = "Hint"
+comment-token = "#"
+indent = {tab-width = 2, unit = " "}
+language-server = {command = "elixir-ls"}
```
Comment (lukaszsamson) [inline README.md:114]: Please add debug adapter as well. Judging from https://helix-editor.com/news/release-22-03-highlights/ it's supported

File: README.md (hunk 1 above)
Comment (samusz) [reply README.md:115]: Sure. I'll test it first and modify the PR once that's done.

**Comment by Wijnand** [comment]
This seems to be the old lsp config format. Since the latest release this format was changed.

**Comment by lukaszsamson** [comment]
Ping

File: README.md (hunk 2 above)
Comment (samusz) [reply README.md:114]: Sorry, I don't have time to get this done now. Hope to get to it soon.

**Comment by dawkrish** [comment]
I recently made a PR here, it may help 
https://github.com/elixir-lsp/elixir-ls/pull/1079

**Comment by samusz** [comment]
Since #1079 is merged, this PR can be closed.

**Pull request closed without merging**